clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -182,12 +182,11 @@ public static synchronized TachyonFS get(TachyonConf tachyonConf) throws IOExcep
   private TachyonFS(TachyonConf tachyonConf) throws IOException {
     super(tachyonConf);
 
-    String masterHost =
-        tachyonConf.get(Constants.MASTER_HOSTNAME,
-            NetworkAddressUtils.getLocalHostName(tachyonConf));
-    int masterPort = tachyonConf.getInt(Constants.MASTER_PORT, Constants.DEFAULT_MASTER_PORT);
-
-    mMasterAddress = new InetSocketAddress(masterHost, masterPort);
+    //String masterHost = NetworkUtils.getMasterHostName(tachyonConf);
+        //tachyonConf.get(Constants.MASTER_HOSTNAME, NetworkUtils.getLocalHostName(tachyonConf));
+    //int masterPort = tachyonConf.getInt(Constants.MASTER_PORT, Constants.DEFAULT_MASTER_PORT);
+    //mMasterAddress = new InetSocketAddress(masterHost, masterPort);
```

Comment:
Are these lines planned to be readded soon? Otherwise I think we should try to avoid having commented out code in the codebase. (Applies to other areas of the code as well)